services/workflows-service/src/workflow/workflow.controller.internal.test.js

```diff
@@ -0,0 +1,159 @@
+const { WorkflowControllerInternal } = require('./workflow.controller.internal');
+const { WorkflowService } = require('./workflow.service');
+const { WorkflowDefinitionModel } = require('./workflow-definition.model');
+const { WorkflowRuntimeDataRepository } = require('./workflow-runtime-data.repository');
+
+const { validate } = require('class-validator');
+
+function _deepCopy(row) {
```

Comment:
consider moving it to `common` and exposing it through `common/test-utils`

Reply:
moved to `@/test-utils`